shared/actions/fs/index.js

```diff
@@ -135,6 +135,20 @@ const setTlfSyncConfig = (state, action) =>
     })
   )
 
+const loadSettings = (state, action) =>
+  RPCTypes.SimpleFSSimpleFSSettingsRpcPromise()
+    .then(settings =>
+      FsGen.createSettingsLoaded({
+        settings: Constants.makeSettings({
+          spaceAvailableNotificationThreshold: settings.spaceAvailableNotificationThreshold,
+        }),
+      })
+    )
+
+const setSpaceNotificationThreshold = (state, action) =>
+  RPCTypes.SimpleFSSimpleFSSetNotificationThresholdRpcPromise({spaceAvailableNotificationThreshold: action.payload.spaceAvailableNotificationThreshold})
+    .then
```

Comment:
did you intend to do something after `then`?

Reply:
Nope nice catch.